yarn-project/foundation/src/fields/fields.ts

```diff
@@ -261,7 +261,7 @@ export class Fr extends BaseField {
       return fromHexString(buf, Fr);
     }
 
-    throw new Error('Tried to create a Fr from an invalid string');
+    throw new Error(`Tried to create a Fr from an invalid string: ${buf}`);
```

Comment:
Improved the error message here as I needed it for debugging issue with slot value parsing.